File: Projeto Php/projeto_completo/src/Criacionais/FactoryMethod/Caixa/BancoCaixaBoleto60Dias.php
<?php

namespace DesignPatterns\Criacionais\FactoryMethod\Caixa;

use DesignPatterns\Criacionais\FactoryMethod\Boleto;

class BancoCaixaBoleto60Dias extends Boleto
{
    public function __construct(float $valor)
    {
        parent::__construct($valor);
        $this->juros = 0.10;
        $this->desconto = 0;
        $this->multa = 0.2;
    }
}

File: Projeto.net/Com.Padroes.De.Projeto/Src/Com.Padroes.De.Projeto.FactoryMethod/2-ProdutoConcreto/Caixa/BancoCaixaBoleto60Dias.cs
using Com.Padroes.De.Projeto.FactoryMethod.ProdutoAbstrado;

namespace Com.Padroes.De.Projeto.FactoryMethod._2_ProdutoConcreto.Caixa
{
    public class BancoCaixaBoleto60Dias : Boleto
    {
        public BancoCaixaBoleto60Dias(float valor) : base(valor)
        {
            Juros = 0.10f;
            Desconto = 0;
            Multa = 0.2f;
        }

    }
}
